articles/spatial-anchors/quickstarts/get-started-ios.md

```diff
@@ -28,7 +28,10 @@ You'll learn how to:
 To complete this quickstart, make sure you have:
 
 - A developer enabled macOS machine with the latest version of <a href="https://geo.itunes.apple.com/us/app/xcode/id497799835?mt=12" target="_blank">Xcode</a> and <a href="https://cocoapods.org" target="_blank">CocoaPods</a> installed.
-- Git installed via HomeBrew. Enter the following command into a single line of the Terminal: `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`. Then, run `brew install git` and `brew install git-lfs`.
+- Git installed via HomeBrew:
+  - Enter the following command into a single line of the Terminal: `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`. 
+  - Then, run `brew install git` and `brew install git-lfs`.
+  - Lastly, update your git config with `git lfs install` (for the current user) or `git lfs install --system` (for the entire system)
```

Comment:
Other lines end in a period. Let's end this one with a period as well to be consistent.